shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/target/UserTargetType.kt

```diff
@@ -0,0 +1,36 @@
+package co.nimblehq.blisskmmic.data.network.target
+
+import co.nimblehq.blisskmmic.BuildKonfig
+import co.nimblehq.blisskmmic.data.network.helpers.TargetType
+import io.ktor.http.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+sealed class UserTargetType: TargetType
+
+class LoginTargetType(email: String, password: String): UserTargetType() {
+
+    @Serializable
+    data class LoginInput(
+        @SerialName("grant_type")
+        val grantType: String,
+        val email: String,
+        val password: String,
+        @SerialName("client_id")
+        val clientId: String,
+        @SerialName("client_secret")
+        val clientSecret: String,
```

Comment:
Can remove the ending `,` 
```suggestion
        val clientSecret: String
```